src/app/components/client/PercentageChart.tsx

```diff
@@ -0,0 +1,132 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use client";
+
+import { CSSProperties } from "react";
+import { useL10n } from "../../hooks/l10n";
+import styles from "./Chart.module.scss";
```

Comment:
Could you also create a separate `PercentageChart.module.scss`? It doesn't really matter that it currently has mostly the same content (though I imagine you can also remove quite a few styles), since future changes are likely to not need to be applied to both, and CSS's cascade has a habit of making changes apply at unintended locations if you don't have a mental model of the entire codebase and its styles.

Reply:
Fixed, thanks